packages/story-editor/src/components/canvas/karma/backgroundCopyPaste.karma.js

```diff
@@ -256,7 +265,11 @@ describe('Background Copy Paste integration', () => {
   async function addBackgroundImage(index) {
     // Add image and click "set as background"
     const image = fixture.editor.library.media.item(index);
-    await fixture.events.click(image);
+    await fixture.events.mouse.seq(({ moveRel, down, up }) => [
+      moveRel(image, 20, 20),
+      down(),
+      up(),
+    ]);
```

Comment:
You can do `clickOn(image, 20, 20)` - at least I think that would work. It takes optional relative position in x and y as positional arguments at least.

So just:

```suggestion
    await fixture.events.mouse.clickOn(image, 20, 20);
```

Reply:
Eh, good point!